meson.build

```diff
@@ -244,6 +244,17 @@ if get_option('enable-tflite-nnapi-delegation')
   # For tf-lite/nnapi, enable-nnfw allows to use nnfw::tflite::nnapi as a backend.
 endif
 
+# ArmNN
+if get_option('enable-armnn')
+  armnn_dep = dependency('armnn', required: true)
```

Comment:
As it forces `required: true`, the below if-statement is unnecessary.

Reply:
Updated.